js/templates/serverConfigForm.html

```diff
@@ -46,21 +46,14 @@
               </div>
             </div>
 
-            <div class="flexCol-8 borderRight0 custCol-border <% ob.errors.server_ip && print('invalid') %>">
+            <div class="flexCol-3 borderRight0 custCol-border <% ob.errors.server_ip && print('invalid') %>">
               <input name="server_ip" type="text" id="fld_server" placeholder="" class="fieldItem" required="" value="<%= ob.server_ip %>">
-              <div class="btn btn-txt btn-corner btn-cornerTR custCol-secondary ion-locked
-                          positionAbsolute tooltip <% if(!ob.SSL){ %> hide <% } %>  js-sslOn"
-                   data-tooltip="<%= polyglot.t('SSLIsOn') %>">
-                <%= polyglot.t('SSLOn') %>
-              </div>
-              <div class="btn btn-txt btn-corner btn-cornerTR custCol-secondary ion-unlocked
-                          positionAbsolute tooltip <% if(ob.SSL){ %> hide <% } %>  js-sslOff"
-                   data-tooltip="<%= polyglot.t('SSLIsOff') %>">
-                <%= polyglot.t('SSLOff') %>
-              </div>
+            </div>
+            <div class="flexCol-5 sslSwitch-radio ustCol-border <% ob.errors.server_ip && print('invalid') %>">
+              <input type="radio" class="fieldItem" id="js-sslOn" name="sslSwitch" value="true" checked /><label for="js-sslOn" class="radioLabel"><%= polyglot.t('SSLOn') %></label>
+              <input type="radio" class="fieldItem" id="js-sslOff" name="sslSwitch" value="false" /><label for="js-sslOff" class="radioLabel"><%= polyglot.t('SSLOff') %></label>
```

Comment:
The state of the radio button isn't being set to match what's in the model on render (that's what `<% if(!ob.SSL){ %> hide <% } %>` did in the existing code). 

You need to add some code around the checked attribute so the input that matches the value in the model is checked, something like: 

`<% if(ob.SSL) { %>checked<% } %> />`